cfme/tests/services/test_provision_stack.py

```diff
@@ -16,6 +16,7 @@
 
 
 pytestmark = [
+    pytest.mark.usefixtures("logged_in"),
```

Comment:
I guess this should not be required. Pete is removing all the logged_in references and logged_in fixture anyway.